### PR TITLE
Kumarabd/feature/graphql

### PR DESCRIPTION
**Pull request by kumarabd**
**Description**

Graphql plugin functionality implemented and tested alongside graphql-playground.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by kumarabd** [comment]
@vineethvanga18 do pull up this branch to add your frontend changes

**Comment by kumarabd** [comment]
@ramrodo looks like a timeout for settings operation. Any idea what exactly the failure means?

File: handlers/extensions.go [hunk 1]
```diff
@@ -50,5 +51,22 @@ func (h *Handler) LoadExtensionFromPackage(w http.ResponseWriter, req *http.Requ
 		extendedEndpoints[output.Router.Path] = output.Router
 	}
 
+	// Run Playground function
+	symRunPlayground, err := plug.Lookup("RunPlayground")
```
Comment (leecalcote) [inline handlers/extensions.go:55]: What's the set of thoughts and intention here? Is there a design spec that speaks to considerations around the playground?

**Review by leecalcote — APPROVED** [review]

**Comment by ramrodo** [comment]
> @ramrodo looks like a timeout for settings operation. Any idea what exactly the failure means?

Right @kumarabd it was only a timeout. But the next workflows went well 👍🏼

File: handlers/extensions.go (hunk 1 above)
Comment (leecalcote) [reply handlers/extensions.go:55]: I created a discussion so as to not miss the need to address this. Can you address before merge?

Comment (leecalcote) [reply handlers/extensions.go:55]: @kumarabd while you're updating Meshery Arch for database storage details, will you add documentation of considerations around this playground?

**Review by leecalcote — CHANGES_REQUESTED** [review]
@kumarabd please note the requests for written consideration of the playground component.

**Review by leecalcote — APPROVED** [review]

File: handlers/extensions.go (hunk 1 above)
Comment (kumarabd) [reply handlers/extensions.go:55]: This was a temporary hack. has been removed now from the master branch and the latest release.

**Pull request merged**